store/mockstore/unistore/cophandler/closure_exec.go

```diff
@@ -837,6 +842,12 @@ func (e *closureExecutor) tableScanProcessCore(key, value []byte) error {
 	if err != nil {
 		return errors.Trace(err)
 	}
+	// Add ExtraPhysTblID if requested
+	// Assumes it is always last!
```

Comment:
I will rework this, so it is not requiring order of handle id, PIDCol or PhysicalTableID. Although I think trying to be consistent with the order of them is a good thing, but it should not be a requirement during reading it.

Reply:
The assumption is that handle_id, pid_id and physical_table_id is last, in that order.

One of the reasons is for index reads, where the column order must be the same as it is stored, see [here](https://github.com/tikv/tikv/blob/8f4f71ea655c5ba794fd2f1780e98e6c3831bc16/components/tidb_query_executors/src/index_scan_executor.rs#L58). But since PhysicalTableID is not explicitly stored, but the already stored TableID in the key-prefix, I assume we could change the requirement to allow it anywhere.
But since it is the last column added by the planner, I think it is OK to keep this.